src/components/communications/Message/Message.tsx

```diff
@@ -49,7 +56,10 @@ export const Message: FunctionComponent<MessageProps> = ({
       : groupWithPrevious
       ? className('bottom')
       : className('single')
-  const isOutgoing = expand(message.sender).id.startsWith('mem_')
+  const sender = expand(message.sender)
+  const isOutgoing = sender?.id.startsWith('mem_')
+  const senderPhoto =
+    sender?.object === 'api_key' ? null : sender.profile_image ? expand(sender.profile_image) : null
```

Comment:
Should we actually error or render nothing if the sender is an API key? A member should never receive a message where the sender is an api key - this only happens for system messages generated by a call where the API key performed the action like updating the thread status or assignee

Reply:
Yeah I think that's why I was comfortable with this approach - members should never see a system generated message. In the unlikely event that they do (which would be a bug in the API), this feels like a sensible fallback.